Lib/glob.py

```diff
@@ -45,6 +45,15 @@ def iglob(pathname, *, root_dir=None, dir_fd=None, recursive=False,
     """
     sys.audit("glob.glob", pathname, recursive)
     sys.audit("glob.glob/2", pathname, recursive, root_dir, dir_fd)
+    # expand ~; see issue 84037
```

Comment:
```suggestion
    # expand ~; see gh-84037
```

Or, since it's an issue prior to the GitHub migration:

```suggestion
    # expand ~; see bpo-39856
```

I'm not sure we need the issue number here at all, though. We generally don't put those for features.